packages/senchabot-prisma/prisma/seed.ts

```diff
@@ -2,11 +2,60 @@ import { PrismaClient } from "@prisma/client";
 
 const prisma = new PrismaClient();
 
+interface AccountId {
+  providerAccountId: string;
+}
+
 async function main() {
-  // Fill this variable with your logged Twitch account id
-  const twitchId = "";
+  const twitchAccount: AccountId | null = await prisma.account.findFirst({
+    where: {
+      provider: "twitch",
+    },
+    select: {
+      providerAccountId: true,
+    },
+  });
+
+  const discordAccount: AccountId | null = await prisma.account.findFirst({
+    where: {
+      provider: "discord",
+    },
+    select: {
+      providerAccountId: true,
+    },
+  });
+
+  const discordServerId = await prisma.discordServer.findFirst({
+    select: {
+      serverId: true,
+    },
+  });
+
+  const twitchChannelId = await prisma.twitchChannel.findFirst({
+    select: {
+      channelId: true,
+    },
+  });
 
-  addCommandsAndAliases(twitchId);
+  if (twitchAccount && !twitchChannelId && discordAccount && !discordServerId) {
+    const twitchAccountId = twitchAccount?.providerAccountId || null;
+    const discordUserId = discordAccount?.providerAccountId || null;
+
+    twitchDataCreate(twitchAccountId);
+    discordDataCreate(discordUserId);
+  } else if (discordAccount && !discordServerId) {
+    const discordUserId = discordAccount?.providerAccountId || null;
```

Comment:
I think `discordUserId` shouldn't be like this.
If we change it like this:
```suggestion
    const discordUserId = discordAccount.providerAccountId;
```
then we won't need to check `discordUserId` in `discordDataCreate`.

We can also do this for `twitchAccountId`.